kge/config-default.yaml

```diff
@@ -123,6 +124,13 @@ dataset:
   # Additional dataset specific keys can be added as needed
   +++: +++
 
+## PLUGINS #####################################################################
+
+# Python packages that should be searched when loading models, embedders, etc.
+# The plugin packages must use the same module hierarchy as LibKGE.
+plugins: [ ]
```

Comment:
No need to enforce the same hierarchy, I think. All packages that contains models or embedders should be listed here.

Reply:
`kge.model` and `kge.model.embedder` should be always included, I think.

Reply:
Yes, I'll rephrase this. I only meant to convey that each plugin package should have at least a `model` and `model.embedder` module. Not that the entire hierarchy should be the same.

Reply:
I was suggesting to not even require this ;) -- just specify the full module that contains the required models and/or embedders.

Reply:
Ah I see, and then include kge.model and kge.model.embedder in the default value, right?

Reply:
I suppose this would be nicest, that would also remove all occurrences of `kge` as a special case in the code.
If we do this, perhaps a better name for the config key would be `modules` instead of `plugins`?

Reply:
Right & I agree, `modules` is more appropriate.